pymc3/sampling.py

```diff
@@ -100,18 +100,19 @@ def assign_step_methods(model, step=None, methods=STEP_METHODS,
     return steps
 
 
-def sample(draws, step=None, init='auto', n_init=200000, start=None,
-           trace=None, chain=0, njobs=1, tune=None, nuts_kwargs=None,
+def sample(draws=500, step=None, init='auto', n_init=200000, start=None,
+           trace=None, chain=0, njobs=1, tune=500, nuts_kwargs=None,
            step_kwargs=None, progressbar=True, model=None, random_seed=-1,
-           live_plot=False, **kwargs):
+           live_plot=False, discard_tuned_samples=True, **kwargs):
     """Draw samples from the posterior using the given step methods.
 
     Multiple step methods are supported via compound step methods.
 
     Parameters
     ----------
     draws : int
-        The number of samples to draw.
+        The number of samples to draw. Defaults to 1000. The number of tuned
```

Comment:
The default is set to 500 in the function definition, not 1000.
How about adding the default to the type:
```
draws : int, default=1000
```
Same for the other parameters.